src/lib/config/uiConfig.js

```diff
@@ -0,0 +1,19 @@
+import _find from "lodash/find";
```

Comment:
for the config overriding, I think we can discuss IRL

Reply:
yes I didn't start with this yet, I copied for now only to make things work.  But yes it needs to be done a bit different